### PR TITLE
Add -html switch handling in case of utf-8 enoded attachment.

### PR DESCRIPTION
**Pull request by iav**
Respect command-line key "-html" if utf8 text attached.
Fix for issue #24.

### CODE REVIEW

File: src/message.c [hunk 1]
```diff
@@ -349,7 +349,12 @@ makeMessage(dstrbuf *in, dstrbuf *out, const char *border, CharSetType charset)
 	if (Mopts.attach) {
 		dsbPrintf(out, "--%s\r\n", border);
 		if (charset == IS_UTF8 || charset == IS_PARTIAL_UTF8) {
-			dsbPrintf(out, "Content-Type: text/plain; charset=utf-8\r\n");
+			if (Mopts.html) {
+				dsbPrintf(out, "Content-Type: text/html; charset=utf-8\r\n");
+			}
```
Comment (deanproxy) [inline src/message.c]: I'd like to see the coding style of the file maintain consistency in any pull request.  Currently the entire file exhibits the K&R brace style.

Comment (iav) [inline src/message.c]: I see only one: line 354  and 355 should be merged. Is it right, or I miss something else?

Comment (deanproxy) [inline src/message.c]: That's correct.  It's a small nit-pick, but I like to maintain consistency as much as possible. :)

**Comment by iav** [comment]
I edit my branch. Should I do a new request?

**Comment by deanproxy** [comment]
I think if you edit your branch, it should show up in the current pull
request.

**Comment by iav** [comment]
Yes. Now there 2 paches. Can they be merged into one?

**Comment by deanproxy** [comment]
There were two commits, but the diff for the merge just see's the last changes. So it merges fine.

**Pull request merged**